### PR TITLE
nifi 1.0 - first version of the documentation

### PR DESCRIPTION
**Pull request by us3r**
Basic documentation about the Nifi installation

### CODE REVIEW

File: 1.8/nifi/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# How to use Apache Nifi on DC/OS
+
+[Nifi](http://nifi.apache.org) is an easy to use, powerful, and reliable system to process and distribute data.
+Running Apache Nifi on DC/OS allows you to manage your data flow very easile. Saving data on the HDFS or consuming data from Kafka will be easily scalable.
```
Comment (mhausenblas) [inline 1.8/nifi/README.md]: `easile` -> `easily` and `the HDFS` -> `HDFS` (drop `the)

File: 1.8/nifi/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- Estimated time for completion: 5 minutes
+- Target audience: Data engineers
+- Scope: Install and use Apache Nifi.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Nifi](#install-nifi)
+- [Use Nifi](#use-nifi)
+- [Uninstall Jenkins](#uninstall-jenkins)
```
Comment (mhausenblas) [inline 1.8/nifi/README.md]: Fix the uninstall (should be nifi and not jenkins), please make sure that the links in the ToC actually work

File: 1.8/nifi/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## Prerequisites
+
+- A running DC/OS 1.8 cluster with at least 1 node.
```
Comment (mhausenblas) [inline 1.8/nifi/README.md]: Please be specific. One public node? One private node?

File: 1.8/nifi/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+![Nifi DC/OS service](img/services.png)
+
+
+## Use Nifi
+
+You typically want to access Nifi via a web browser outside of the DC/OS cluster. To access the Apache Nifi UI from outside of the DC/OS cluster you can use [Marathon-LB](https://dcos.io/docs/1.8/usage/service-discovery/marathon-lb/), which is recommended for production usage.
+
+You can also use [Admin Router](https://dcos.io/docs/1.8/development/dcos-integration/#-a-name-adminrouter-a-admin-router) to provide access to the Apache NIfi UI, which is fine for dev/test setups.
+
+In the following we will use the Endpoint URL.
```
Comment (mhausenblas) [inline 1.8/nifi/README.md]: Which Endpoint URL?

File: 1.8/nifi/README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+You can also use [Admin Router](https://dcos.io/docs/1.8/development/dcos-integration/#-a-name-adminrouter-a-admin-router) to provide access to the Apache NIfi UI, which is fine for dev/test setups.
+
+In the following we will use the Endpoint URL. 
+
+ - Go to the **Services** tab
+ - Select "nifi" in the list of running services
+ - Once the Nifi service is `Healthy`,
+   Select the "nifi" task.
+ - Click the Endpoint URL to open the Calico status page in a new tab.
```
Comment (mhausenblas) [inline 1.8/nifi/README.md]: Calico?

**Comment by mhausenblas** [comment]
Thanks a lot for this contrib @us3r! I left a few comments, essentially editorial changes (grammar, typos, inconsistencies, etc.) and would also like to ask you to add a section on what can be done with Nifi, see as an example what we're doing with [Kafka](https://github.com/dcos/examples/tree/master/1.8/kafka#produce-and-consume-messages).

**Comment by us3r** [comment]
Thanks @mhausenblas ! Could you be so kind and check once again? Sorry but as you probably noticed english is not my native language ...so some typos can be still there.

**Comment by mhausenblas** [comment]
That's awesome, thanks a lot @us3r! LGTM and now we just need to wait until the Universe PR is merged to 🚢 it! ;)

**Comment by ryadav88** [comment]
@mhausenblas Universe PR is merged.

**Comment by mhausenblas** [comment]
Thanks @ryadav88!

**Pull request merged**